### ISSUE TITLE
Decide if historical budget data is useful for MVP

### ISSUE THREAD

**Issue by meganmckissack** [hackoregon/team-budget#19]
Online budget data goes back to Brass Reports for FY 2005-06
- [ ] are service areas consistent in all years
- [ ] does it align with MVP, what data is usefull

**Comment by MikeTheCanuck** [comment]
What specifically would allow us to determine whether it's worth the effort of pulling in additional years' data into the CSVs (and later to the database)?

I don't believe that further years' data is necessary for us to have at this point - we are currently blocked on frontend development, and until we can start to render some prototype pages/cards for the Budget in Brief and/or other stories to be articulated, additional data is redundant.

As soon as we start seeing the shape of the frontend designs in working code, we'll have a much better idea how much more important it will be (very important? nice to have?) to get 3+ years of historical data coming back from the API endpoints.

**Comment by MikeTheCanuck** [comment]
Jim suggested tonight he might have time to pull in an additional year's data this week.  Jim, I've temporarily assigned this to you as a reminder to note any findings you make in that work.

**Comment by jimtyhurst** [comment]
I think it will be useful to have at least 3 years of data to display, although I don't think that is strictly required for the MVP. The `team-budget` repository already has data for fiscal years 2015-16 and 2016-17 extracted from "Budget in Brief" documents.

At the weekly meeting on 2017-02-27, a few people divided up additional "[Budget in Brief](https://drive.google.com/drive/folders/0B04ORgI23dk7MmFfYURrN05LTms)" documents going back to fiscal year 2012-13, which would give us 5 years of data. During the next week, people will work to extract and transform the data. @meganmckissack demonstrated how to use the [`tabula`](http://tabula.technology/) tool to extract the data easily from a PDF file to a CSV file. But the difficult part is to transform the data from those tables into the "[tidy](https://www.jstatsoft.org/article/view/v059i10)" format of the existing `Budget_in_Brief_OCRB_data_All_Years.csv` file. Hopefully, one of us will come up with a script to do that transformation. @maryannet pointed me to the R libraries [`tidyr` and `dplyr`](https://rpubs.com/bradleyboehmke/data_wrangling), which have functions for doing just these types of transformations.

**Comment by jimtyhurst** [comment]
Closed by the merge of Pull Request #60. `master` branch now has a `history` endpoint for querying the past 10 years of data.
